js/test/unit/ipc/writer/stream-writer-tests.ts

```diff
@@ -55,6 +64,38 @@ describe('RecordBatchStreamWriter', () => {
         writer.close();
         await validate;
     });
+
+    it('should write delta dictionary batches', async () => {
```

Comment:
It looks like there's no corresponding "should read delta dictionary batches" test (unless I'm missing something). Would that be hard to add?

Reply:
That case should be covered a few lines below here, on this line:
https://github.com/apache/arrow/blob/6a70a25bdfc22958f2dced8623cad594d41dc059/js/test/unit/ipc/writer/stream-writer-tests.ts#L91

Reply:
Ahh yeah, my bad.